### PR TITLE
feat: add network to voiting info

### PR DESCRIPTION
**Pull request by MareskoY**
## General Changes

Screenshot 2024-01-11 at 15 33 23

### CODE REVIEW

File: src/modules/governance/proposal/VoteInfo.tsx [hunk 1]
```diff
@@ -42,9 +47,45 @@ export function VoteInfo({ proposal }: VoteInfoProps) {
...
+            {network && (
+              <Box
+                sx={{
+                  display: 'flex',
+                  alignItems: 'center',
+                  color: 'text.secondary',
+                }}
+              >
+                <Typography variant="caption">
+                  <Trans>This vote is free on</Trans>
```
Comment (grothem) [inline src/modules/governance/proposal/VoteInfo.tsx:66]: We currently are not using the relayer for gasless voting. Maybe for now we could just change this text to `Voting is on` and then we can update later.

**Review by grothem — CHANGES_REQUESTED** [review]

File: src/modules/governance/proposal/VoteInfo.tsx (hunk 1 above)
Comment (foodaka) [reply src/modules/governance/proposal/VoteInfo.tsx:66]: @MareskoY can you update we can get this into main

**Comment by foodaka** [comment]
@MareskoY can u fix the conflicts

**Review by foodaka — CHANGES_REQUESTED** [review]
needs to fix conflicts

**Review by foodaka — APPROVED** [review]

**Review by grothem — APPROVED** [review]

**Pull request merged**